cypress/component/ChromeRoutes/ChromeRoute.cy.tsx

```diff
@@ -75,20 +105,36 @@ const Wrapper = ({ children, getUser }: React.PropsWithChildren<{ getUser?: () =
 
   return (
     <IntlProvider locale="en">
-      <ScalprumProvider scalprum={scalprum.current}>
-        <JotaiProvider>
-          <BrowserRouter>
-            <Routes>
-              <Route path="*" element={children} />
-            </Routes>
-          </BrowserRouter>
-        </JotaiProvider>
-      </ScalprumProvider>
+      <ChromeAuthContext.Provider value={chromeAuthContextValue}>
+        <ScalprumProvider scalprum={scalprum.current}>
+          <JotaiProvider>
+            <BrowserRouter>
+              <FeatureFlagsProvider>
+                <Routes>
+                  <Route path="*" element={children} />
+                </Routes>
+              </FeatureFlagsProvider>
+            </BrowserRouter>
+          </JotaiProvider>
+        </ScalprumProvider>
+      </ChromeAuthContext.Provider>
     </IntlProvider>
   );
 };
 
 describe('ChromeRoute', () => {
+  beforeEach(() => {
+    cy.intercept('GET', '/api/featureflags/*', {
+      toggles: [
+        {
+          name: 'platform.virtual-assistant.is-open-config',
+          enabled: true,
+          variant: { name: 'disabled', enabled: true },
+        },
+      ],
+    });
+  });
+
   it('should render not found route if permissions are not met', () => {
```

Comment:
Just to make sure I understand the test scenario. Would this cover something like a non-admin trying to view `/iam/users` which is restricted to org admins or users with appropriate assigned permissions?